baldrick/plugins/circleci_artifacts.py

```diff
@@ -11,6 +14,7 @@ def set_commit_status_for_artifacts(repo_handler, payload, headers):
         return "Skipping artifact check, disabled in config."
 
     if payload['status'] == 'success':
+        LOGGER.info(r"Got successful call for repo: %s/%s", payload['username'], payload['reponame'])
```

Comment:
Tsk tsk... no f-string?

Reply:
This is the "correct" way to use logging, even if I don't particularly like it. Logging does the substitution internally. 🙄